### PR TITLE
Release + prerelease notes for Edge 97/99

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+ICoreWebView2ExperimentalFrame2
+ 
+(ADDED)
+     PostWebMessageAsJson
+     PostWebMessageAsString
+     add_WebMessageReceived
+     remove_WebMessageReceived
+ICoreWebView2ExperimentalFrameWebMessageReceivedEventHandler
+ (ADDED)
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/release-notes.md:66]: @peiche-jessica - this experimental idl diff showed up in the release section of the script. Not sure why

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+#### Promotions
+  
+The following items are now stable:
+*  The [Media API](/microsoft-edge/webview2/reference/win32/icorewebview2experimental9?view=webview2-1.0.1083-prerelease&preserve-view=true#summary) that enables developers to mute/unmute media within the WebView.
+*  The [Download Positioning and Anchoring API](/microsoft-edge/webview2/reference/win32/icorewebview2experimental11?view=webview2-1.0.1083-prerelease&preserve-view=true).  This API enables:
+   *  Changing the position of the download dialog, relative to the WebView2 bounds.  You can anchor the download dialog to the **Download** button, instead of the default position, which is the top-right corner.
+   *  Programmatically opening and closing the default download dialog.
+   *  Making changes in response to the dialog opening and closing.
+
+//Ask @Jessica
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:54]: Change code comment "//" to markdown comment ""

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+  
+[NuGet package for WebView2 SDK 1.0.1133](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1133-prerelease)  
+  
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 99.0.1133.0 or higher.  
+
+### General
+
+#### Experimental Features
+
+* Added support for themeing:
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:81]: I believe "themeing" should be spelled "theming" ... but if so we also have a mistake in the TOC at https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/react-native/theme-config

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+* The following [APIs for iframes](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe2?view=webview2-1.0.1083-prerelease&preserve-view=true) in WebView2:
+   *  `PostWebMessageAsJson`
+   *  `PostWebMessageAsString`
+   *  `add_WebMessageReceived`
+   *  `remove_WebMessageReceived`
+
+* The ProcessInfo APIs which provides more information about WebView2 [processes](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprocessinfo?view=webview2-1.0.1083-prerelease&preserve-view=true) and [process collections](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprocessinfocollection?view=webview2-1.0.1083-prerelease&preserve-view=true).
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:121]: Change "which provides" to just "provide"

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
+
+## 1.0.1071.54
```
Comment (oggy22) [inline microsoft-edge/webview2/release-notes.md]: 1072

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+  
+Release Date: January 10, 2022
```
Comment (oggy22) [inline microsoft-edge/webview2/release-notes.md:37]: 13

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
+
+## 1.0.1071.54
+  
+Release Date: January 10, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1071.54](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1071.54-release)
```
Comment (oggy22) [inline microsoft-edge/webview2/release-notes.md]: 1072

**Comment by jasonstephen15** [comment]
@Reezaali / @mikehoffms - I think this is good to go. I have commented out the prerelease section for now, while we wait for generation of ref docs. @vchapel has done his edit pass, and I sign off.

**Comment by jasonstephen15** [comment]
@vchapel mind approving this? we need an approving review before being able to approve.

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -28,9 +28,100 @@ WebView2 bug fixes are either Runtime-specific or SDK-specific.
...
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 97.1072.54 or higher.  
+
+### General
+
+#### Promotions
+  
+The following items are now stable:
+*  The [Media API](/microsoft-edge/webview2/reference/win32/icorewebview2_8?view=webview2-1.0.1072.54&preserve-view=true#summary) that enables developers to mute/unmute media within WebView2.
+*  The [Download Positioning and Anchoring API](/microsoft-edge/webview2/reference/win32/icorewebview2_9?view=webview2-1.0.1072.54&preserve-view=true).  This API enables:
```
Comment (Reezaali) [inline microsoft-edge/webview2/release-notes.md:49]:
```suggestion
*  The [Download Positioning and Anchoring API](/microsoft-edge/webview2/reference/win32/icorewebview2_9?view=webview2-1.0.1072.54&preserve-view=true) enables:
```

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -28,9 +28,100 @@ WebView2 bug fixes are either Runtime-specific or SDK-specific.
...
-To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
+To use a prerelease SDK along with a Microsoft Edge preview channel, go to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
+
+## 1.0.1072.54
+  
+Release Date: January 13, 2022
+  
+[NuGet package for WebView2 SDK 1.0.1072.54](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1072.54-release)
```
Comment (Reezaali) [inline microsoft-edge/webview2/release-notes.md:39]: Is the link correct?

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -28,9 +28,100 @@ WebView2 bug fixes are either Runtime-specific or SDK-specific.
...
+### General
+
+#### Promotions
+  
+The following items are now stable:
+*  The [Media API](/microsoft-edge/webview2/reference/win32/icorewebview2_8?view=webview2-1.0.1072.54&preserve-view=true#summary) that enables developers to mute/unmute media within WebView2.
+*  The [Download Positioning and Anchoring API](/microsoft-edge/webview2/reference/win32/icorewebview2_9?view=webview2-1.0.1072.54&preserve-view=true).  This API enables:
+   *  Changing the position of the download dialog, relative to the WebView2 bounds.  You can anchor the download dialog to the **Download** button, instead of the default position, which is the top-right corner.
+   *  Programmatically open and close the default download dialog.
+   *  Make changes in response to the dialog opening and closing.
```
Comment (Reezaali) [inline microsoft-edge/webview2/release-notes.md:52]:
```suggestion
   *  Making changes in response to the dialog opening and closing.
```

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -28,9 +28,100 @@ WebView2 bug fixes are either Runtime-specific or SDK-specific.
...
+*  The following [APIs for iframes](/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1072.54&preserve-view=true) in WebView2:
```
Comment (Reezaali) [inline microsoft-edge/webview2/release-notes.md:53]:
```suggestion
*  New [APIs for iframes](/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1072.54&preserve-view=true):
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Review by Reezaali — APPROVED** [review]

**Pull request merged**